HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -17,6 +17,19 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+
+def customiseForPR22293(process):
+    for producer in producers_by_type(process, "HBHEPhase1Reconstructor"):
+        producer.algorithm.chiSqSwitch = cms.double(15.0)
+        producer.saveEffectivePedestal = cms.bool(True)
+        if (producer.algorithm.useMahi == cms.bool(True)):
+            producer.algorithm.applyTimeConstraint = cms.bool(False)
+            producer.algorithm.applyPedConstraint = cms.bool(False)
+        if producer.algorithm.timeSlewPars:
```

Comment:
I think this should be `if hasattr(producer.algorithm,"timeSlewPars"):`

Reply:
fixed